scikit_learn_script_mode_local_training_and_serving/scikit_learn_script_mode_local_training_and_serving.py

```diff
@@ -16,40 +16,54 @@
 import os
```

Comment:
This is a sample Python program that trains a simple scikit-learn model on the Iris dataset. -> This is a sample Python program that trains a simple scikit-learn model on the California dataset.